io-engine/src/grpc/v1/nexus.rs

```diff
@@ -1113,3 +1124,20 @@ impl NexusRpc for NexusService {
             .map(Response::new)
     }
 }
+
+// Nexus event messages from Nexus data.
+impl<'n> Event for nexus::Nexus<'n> {
+    fn event(&self, event_action: Option<EventAction>) -> EventMessage {
```

Comment:
Do we need Option to denote event action ? Can we even have event_action as None ?

Reply:
We need not pass EventAction for the some struct other than nexus::Nexus<'n> which has a single EventAction. But there are no events for now which do not need the EventAction to be passed. And also we need not have optional if we pass EventAction for all events. I will remove the optional.